### ISSUE TITLE
Missing css class="form-control" on Date Field input

### ISSUE THREAD

**Issue by ankurk91** [kevinchappell/formBuilder#351]
Hi,
The form builder have a bootstrap ```form-control``` css class on all input fields except **Date Field**

This is the html generated for Date input -
```
<input type="date" class="calendar" name="date-1483767794498-preview" id="date-1483767794498-preview">
```

See screenshot, taken from [this](http://formbuilder.readthedocs.io/en/latest/formBuilder/options/showActionButtons/) page

![screenshot from 2017-01-07 11-13-57](https://cloud.githubusercontent.com/assets/6111524/21739770/f46f8c46-d4ca-11e6-8c2a-e7a7d95d69b3.png)

**Comment by ankurk91** [comment]
@kevinchappell 
Hi, seems like the referenced commits above was not included in most recent release (v1.24.5).
Above patch works well btw. 
Right now i am doing a lot of hacks to add that css class. Please merge that commit.

Thanks.

**Comment by ankurk91** [comment]
Many thanks for releasing 1.24.6 with the fix.
